Makefile

```diff
@@ -7,3 +7,9 @@ bench:
 cover:
 	@go test -coverprofile=coverage.out -covermode=count
 	@go tool cover -html=coverage.out -o coverage.html
+
+lint:
+	@golangci-lint run --new-from-rev=e0a5614e47d349897~0
```

Comment:
hmm, can we not fix all issues instead? Or what amount of issues are we looking at

Reply:
There's quite a lot of issues, like 30+ that would require extensive testing. ofc it can be done but I've used the `new-from-rev` successfully in the past. Just whenever you're bored, Move increase the number 0 by 1 and you'll start linting one commit further back in history.